src/js/tabular_comparison/result.js

```diff
@@ -0,0 +1,250 @@
+import React, {useEffect, useState} from "react";
+import {Card, Grid, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow} from "@mui/material";
+import {format as d3format} from "d3-format";
+import {scaleSequential as d3scaleSequential} from 'd3-scale';
+import {defaultValues} from "../defaultValues";
+import {fillMissingKeys} from "../utils";
+
+//components
+import SectionTitle from "./section-title";
+
+const Result = (props) => {
+    const [rows, setRows] = useState([]);
+    const arrowSvg =
```

Comment:
It will be good if we can move svg icons in separate files and import from there.

Reply:
good point, done